### PR TITLE
Elaborated installation instruction

### PR DESCRIPTION
**Pull request by Ayush-iitkgp**
Hi,
I had the relatively tough time trying to get SCIP.jl working. This attempt is to make installing SCIP.jl easier for people who don't understand how to set ENV variables and including missing steps during installation. Also I got SCIP.jl working without copying the [patched file](http://scip.zib.de/download/bugfixes/scip-3.2.1/Makefile.doit), I think this step is no longer required.

### CODE REVIEW

**Comment by leethargo** [comment]
Thanks for the effort, I'll have a look later.
1. The failing test is most likely not caused by your changes, I think it has to do with changes in the JuMP tests that we use.
2. I'd be surprised if the patch for SCIP is no longer needed, since they didn't do a release for it. I guess it's not strictly necessary for SCIP.jl to work at all, but it was necessary for something in particular, maybe linking to a `libscipopt.so` that expects IPOPT.

File: README.md [hunk 1]
```diff
@@ -20,23 +20,27 @@ Julia interface to [SCIP](http://scip.zib.de) solver.
 
 ## Installation
 
-The SCIP.jl package requires SCIP to be installed.
+The SCIP.jl package requires [SCIP](http://scip.zib.de/) to be installed.
 
 [Download](http://scip.zib.de/download.php?fname=scipoptsuite-3.2.1.tgz) the
-SCIP Optimization Suite, copy the
-[patched file](http://scip.zib.de/download/bugfixes/scip-3.2.1/Makefile.doit)
```
Comment (leethargo) [inline README.md]: Maybe the story of the patch should be moved somewhere else, together with the reasoning for it (IPOPT support?).

File: README.md [hunk 2]
```diff
@@ -20,23 +20,27 @@ Julia interface to [SCIP](http://scip.zib.de) solver.
...
+```
+CSIP needs the library in `${SCIPOPTDIR}/lib/scipoptlib.so` and the C header files in `${SCIPOPTDIR}/scip-*/src/`.
+
+Clone the SCIP github repository in your local machine and build it in Julia with
 
-Now you should be able to build SCIP.jl in Julia with
 ```
+Pkg.clone("https://github.com/SCIP-Interfaces/SCIP.jl.git")
 Pkg.build("SCIP")
 ```
```
Comment (leethargo) [inline README.md]: Actually, SCIP.jl was registered in `METADATA.jl` since these instructions were written, so could be simplified now.

Comment (Ayush-iitkgp) [inline README.md]: Nice, then we can replace Pkg.build("SCIP") with Pkg.add("SCIP").

File: README.md (hunk 1 above)
Comment (Ayush-iitkgp) [inline README.md]: Actually, I got an error when I replaced the original Makefile with the above patched file (this is what we were supposed to do.right ?)

I will add your point in the **Notes** below the original instruction.

Comment (leethargo) [inline README.md]: The best (most up-to-date) installation instructions are those that are used in the [Travis build](https://github.com/SCIP-Interfaces/SCIP.jl/blob/master/.travis.yml) :wink:

Comment (Ayush-iitkgp) [inline README.md]: Right :D

**Comment by Ayush-iitkgp** [comment]
@leethargo this attempt is inspired from instructions given in Travis Build file. Tried to state the steps in ordered list with explanation of what each step does.

**Pull request merged**